swarm/api/routes/autopilot_routes.py

```
@@ -0,0 +1,470 @@
+
+class AutopilotStopRequest(BaseModel):
+    """Request to stop an autopilot run gracefully."""
+
+    reason: str = Field("user_initiated", description="Reason for stopping")
+
+
+# Global autopilot controller (initialized on first use)
+_autopilot_controller = None
+
+
+def _get_autopilot_controller():
```

Ruff (0.14.13)

Missing return type annotation for private function `_get_autopilot_controller`

(ANN202)